cogs/search.py

```diff
@@ -0,0 +1,266 @@
+import os
+import re
+from typing import List
+from urllib.parse import quote_plus
+from .utils import constants
+import textwrap
+
+import aiohttp
+import html2text
+from discord import Embed, utils
+from discord.ext.commands import Bot, CheckFailure, Cog, CommandNotFound, Context, command, Error
+
+from .search_utils import searchexceptions
+from .search_utils.regex import filter_words
+
+
+class Search(Cog, name="Basic"):
+    """Searches the web for a variety of different resources."""
+    def __init__(self, bot: Bot) -> None:
+
+        # Main Stuff
```

Comment:
```suggestion
class Search(Cog, name="Basic"):
    """Searches the web for a variety of different resources."""
    
    def __init__(self, bot: Bot) -> None:
        # Main Stuff
```
Please, once again, don't mark this as resolved, until it's resolved, if you have probelms with it, comment, don't just mark it as resolved.